gitspy/RepositoryCollectionViewCell.swift

```diff
@@ -10,7 +10,10 @@ import UIKit
 
 class RepositoryCollectionViewCell: UICollectionViewCell {
     
-    @IBOutlet weak var repositoryTitleLabel: UILabel!
+    @IBOutlet private weak var repositoryTitleLabel: UILabel!
     
+    func setRepository(repository: Repository?) {
+        repositoryTitleLabel.text = repository?.name
+    }
```

Comment:
oh okay, what's the preferred way for you? I usually prefer this rather than having long `...cellForRow:` and I don't like to expose labels or views since it's not easy then to refactor/re-design `UI***Cell`(s). Do you have better ways?